service/controller/app/v1/resource/chart/create_test.go

```diff
@@ -98,22 +96,20 @@ func Test_Resource_newCreateChange(t *testing.T) {
 		},
 	}
 
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			c := Config{
-				G8sClient:  fake.NewSimpleClientset(),
-				K8sClient:  k8sfake.NewSimpleClientset(),
-				KubeConfig: kubeconfigtest.New(kubeconfigtest.Config{}),
-				Logger:     microloggertest.New(),
+	c := Config{
+		G8sClient: fake.NewSimpleClientset(),
+		Logger:    microloggertest.New(),
```

Comment:
kubeconfig and k8s client are not used because the controller context is now used. 

Also moves the resource creation out of the test loop.